Source/Rive/Private/Rive/RiveTexture.cpp

```diff
@@ -12,7 +12,7 @@
 
 URiveTexture::URiveTexture()
 {
-    SRGB = true;
+    SRGB = false;
```

Comment:
Has this change been tested on other platforms (e.g.: macOS)? Previously, macOS required specific sRGB settings (for example, the inclusion of the flag in .SetFlags below, but I'm not sure if this specific boolean in the texture will have any affect.

Reply:
I have not tested in any environment other then my laptop. I will remove this for now just in case.